tools/sideloading/README.md

```diff
@@ -0,0 +1,226 @@
+## Overview
+
+**Sideloading** tool is a sample solution to add side-loading capability to your Panorama application. With this solution, you can modify scripts & models, transfer them to the device, and run the updated scripts without re-deploying the entire application. It gives you quicker development iteration experience.
+
+This solution comes with 1) sideloading agent module you can import in your Panorama application, 2) sideloading commandline interface you can use on your development host.
+
+## Features
+
+* Automatically identify updated files and transfer them. (**"sync" command**)
+* Run the updated scripts and models as a process. (**"run-app" command**)
+* Kill the process. (**"kill-app" command**)
+* Easily switch the application from side-loading enabled development mode to production mode, with single line of code change.
+
+
+## Security notice
+
+* > We do not recommend doing this on a production system because this creates a security vulnerability and might disrupt the normal operation of your device.
+* > Opening a port to create a webserver could but doesn't necessarily create a security vulnerability and you will be relying on the security of the downloaded environment to protect your development environment.
+
+
+## Security best practices
+
+* > This sideloading solution uses certificate files and private key files for mutual authentication. Please keep the them secure, use them only for sideloading to your device, and don't share them widely.
+* > When creating cerficiate & private key files, use the appropriate expiry (default : 30 days) and avoid using unncessary long expiry. As needed, please implement peridic rotation of the certificate and key files so that they don't expire.
+* > When you deploy your application to Production environment, close the Inbound networking port for sideloading. Also delete the certificate files and key files.
+
+
+## How to use
+
+1. Under the top directory of your application (where assets / graphs / packages directory exist), run the "install.py" script to generate and install necessary files.
+
+    ``` bash
+    python3 aws-panorama-samples/tools/sideloading/install.py --expiry 30 --app-src-dir ./packages/123456789012-code-1.0/src
```

Comment:
> aws-panorama-samples/tools/sideloading/install.py
> ./packages/123456789012-code-1.0/src

These paths imply that you've cloned this repo into your application folder. I think it's more likely that the application and samples folders would be siblings.